packages/grafana-ui/src/utils/storybook/withTheme.tsx

```diff
@@ -16,20 +16,19 @@ const ThemeableStory: React.FunctionComponent<{ handleSassThemeChange: SassTheme
 
   handleSassThemeChange(theme);
 
+  const css = `#root {
+    width: 100%,
+    padding: 20px,
+    display: flex,
+    minHeight: 100%,
+    background: ${theme.colors.background.primary}
+  }`;
```

Comment:
```suggestion
  const css = `#root {
    width: 100%;
    padding: 20px;
    display: flex;
    min-height: 100%;
    background: ${theme.colors.background.primary};
  }`;
```
Since this is a css declaration now, the syntax needs to be slightly updated, otherwise the styles aren't applied.

Reply:
Thx 👍